### PR TITLE
Migrate from Travis to Github Actions

### PR DESCRIPTION
**Pull request by dmitry-mukhin**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install flake8 pytest
+        pip install 'requests>=1.0' 'python-dateutil>=2.1' 'six>=1.2.0'
+        pip install pytest-cov python-coveralls
```
Comment (mikhail-iurkov) [inline .github/workflows/test.yml:29]: Declare test dependencies in `setup.py` `extras_require` clause, then do `pip install '.[dev]'` here.

This is also useful for local development

File: tests/integration/test_api_resources.py [hunk 2]
```diff
@@ -6,7 +6,13 @@
...
 import time
 
-import mock
+import six
+
+if six.PY3:
+    from unittest import mock
+else:
+    import mock
+
```
Comment (mikhail-iurkov) [inline tests/integration/test_api_resources.py:15]: Maybe something like `from tests.utils import mock`, and move repeating logic there

File: .github/workflows/test.yml (hunk 1 above)
Comment (dmitry-mukhin) [reply .github/workflows/test.yml:29]: the goal of this PR is to move logic from travis more or less intact
your suggestions are nice but I think should be done in separate PR.

File: tests/integration/test_api_resources.py (hunk 2 above)
Comment (dmitry-mukhin) [reply tests/integration/test_api_resources.py:15]: nice, will do

File: .github/workflows/test.yml [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+      run: |
+        pytest tests/functional --cov=pyuploadcare
+
+  django:
+    needs: functional
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # https://docs.djangoproject.com/en/dev/faq/install/#what-python-version-can-i-use-with-django
+        # this is a bit excessive, probably should run only before
```
[the remark this answers is not in the document]
Comment (dmitry-mukhin) [reply .github/workflows/test.yml:74]: it was copied from travis and updated due to current django-python matrix

File: .github/workflows/test.yml [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+    - name: Test with pytest
+      run: |
+        pytest tests/functional --cov=pyuploadcare
+
+  django:
+    needs: functional
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # https://docs.djangoproject.com/en/dev/faq/install/#what-python-version-can-i-use-with-django
```
Comment (homm) [inline .github/workflows/test.yml:73]:
```suggestion
        # https://docs.djangoproject.com/en/3.2/faq/install/#what-python-version-can-i-use-with-django
```

File: .github/workflows/test.yml [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+        # this is a bit excessive, probably should run only before
+        #   releasing new version
+        python-version: [3.6, 3.7, 3.8, 3.9]
+        django-version: [2.2, 3.0, 3.1, 3.2]
+        include:
+          - python-version: 2.7
+            django-version: 1.11
+          - python-version: 3.7
+            django-version: 1.11
```
Comment (homm) [inline .github/workflows/test.yml:82]: > 1.11   |   2.7, 3.4, 3.5, 3.6, 3.7

So 3.5 and 3.6 are missed.

File: .github/workflows/test.yml [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install flake8 pytest
+        pip install 'requests>=1.0' 'python-dateutil>=2.1' 'six>=1.2.0'
+        pip install pytest-cov python-coveralls
+        pip install -e .
```
Comment (homm) [inline .github/workflows/test.yml:30]: Nothing from this list is required except `pip install flake8`

File: .github/workflows/test.yml [hunk 7]
```diff
@@ -0,0 +1,133 @@
...
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install flake8 pytest
+        pip install 'requests>=1.0' 'python-dateutil>=2.1' 'six>=1.2.0'
+        pip install pytest-cov python-coveralls
```
Comment (homm) [inline .github/workflows/test.yml:58]: `flake8` is not required. Others installed with `pip install -e .`

```suggestion
        pip install pytest pytest-cov python-coveralls
```

File: .github/workflows/test.yml [hunk 8]
```diff
@@ -0,0 +1,133 @@
...
```
Comment (homm) [inline .github/workflows/test.yml:96]: `flake8` is not required. Others installed with `pip install -e .`

```suggestion
        pip install pytest pytest-cov python-coveralls
```

**Review by homm — CHANGES_REQUESTED** [review]

File: .github/workflows/test.yml (hunk 5 above)
Comment (dmitry-mukhin) [reply .github/workflows/test.yml:80]: 3.5 has reached EOL, will add 3.6

**Review by homm — APPROVED** [review]
I'd rather return `pip install -U pip`, otherwise is OK

**Pull request merged**